.ci/linux-steps.yaml

```diff
@@ -34,6 +34,16 @@ steps:
       sudo tar -C /opt/ -xvpf julia-1.3.0-linux-x86_64.tar.gz
     fi
 
+    if [ 'a$(R.version)' != 'a' ]; then
+      sudo add-apt-repository 'deb https://cloud.r-project.org/bin/linux/ubuntu xenial-cran40/'
+      sudo apt-get -y update
+      sudo apt-get remove -y r-base-core
```

Comment:
```suggestion
```
I agree with @eddelbuettel. No need to remove `r-base-core`.

Reply:
Removed.